docs/articles/documentation/reference/test-api/requesthook/README.md

```diff
@@ -103,3 +103,33 @@ async onResponse (event) {
     throw new Error('Not implemented');
 }
 ```
+
+### Mock Response Headers
```

Comment:
[can be ignored] I'm not quite sure, but I feel that the "stubbing" term is more suitable here. From [Mocks Aren't Stubs](https://martinfowler.com/articles/mocksArentStubs.html):
> Mocks are what we are talking about here: objects pre-programmed with expectations which form a specification of the calls they are expected to receive. 

> Stubs provide canned answers to calls made during the test, usually not responding at all to anything outside what's programmed in for the test.

Reply:
You are right, this is indeed a stub, not a mock